po/README.md

```diff
@@ -1,22 +1,169 @@
+
+2. Also make sure, that you know some basics about how to make pull requests: [How to make pull request from fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) and [How to create fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks).
+
+3. Upload changes that you make in your local copy of repository to your Github fork.
+
+If you already know how to use git, this can be done like this:
+```bash
+cd extension-manager
+git commit -m "I translated Extension Manager!"
+git push
+```
+
+If not, you could use Github interface to do so:
+
+1. Open your fork on Github, navigate to `./po` folder.
+2. Click `Add file` then `Upload files` and pick your translated `po` file and `LINGUAS`.
+3. Make sure that everything correct, there no typos in translation, mistakes, you didn't upload wrong files by accident, etc.
+4. Open [pull request](https://github.com/mjakeman/extension-manager/compare) to Extension Manager repository against your fork.
+
+## Tips about translating
+1. Try to be consistent in your translation with other GTK/GNOME applications.
+
+Pick same terms, use same accelerations, etc.
+
+2. Always check your translation before opening pull request, don't just blindly translate.
+
+3. Open your `po` file with any text editor, not with POEdit, and in top most of file you will find some placeholder info for your translation. Please, fill it, if you can. Take as example `uk.po` file.
+
+4. `translator-credits` is optional and you not comfortable with sharing your alias or email or link, then leave it as-is. Also, if there will be credits for other translators, please, don't remove them. Just add your credit info on newline.
+
+If you want to put email, do this:
+
+`your_alias <somecooladress@mail.com>`
+
+And if you want to place link:
+
+`your_alias https://example.com`
+
+You can check if you input everything correctly in `About` dialog of Extension Manager.
 
 ## Regenerate POT files
-If you want to regenerate the POT file (i.e. when it hasn't been updated in
+If you want to regenerate the `pot` file (i.e. when it hasn't been updated in
 a while), there are a few steps that need to be followed:
 
-### Update POTFILES with latest source files
-Replace the contents of `POTFILES` with the output of the `print-source-files.sh`
-script. Make sure to run it from this directory (`./po`).
+GNOME Builder version:
+
+1. Open GNOME builder.
+2. Open `New Build Terminal` with `Shift+Ctrl+Alt+T` or click `plus button` in top-left corner and pick `New Build Terminal`
+3. Go to `po` directory and run `print-source-files.sh`.
 
-### Build POT file
-Go to the build directory (typically `_build`, but whichever you specified
-when running meson).
+```bash
+cd extension-manager/po
+./print-source-files.sh
+```
+4. You will get output. Replace contents of `POTFILES` with that output.
```

Comment:
I don't have a strong opinion here, but it might help to avoid some copy/paste errors.

```suggestion
./print-source-files.sh > POTFILES
```

Reply:
[e2b2a6a](https://github.com/mjakeman/extension-manager/pull/514/commits/e2b2a6a07666ae78256afdbf091e102c3a73d369)